octodns_edgedns/__init__.py

```diff
@@ -25,10 +25,10 @@ def __init__(self, resp):
 
 class AkamaiClient(object):
     '''
-    Client for making calls to Akamai Fast DNS API using Python Requests
+    Client for making calls to Akamai Edge DNS API using Python Requests
```

Comment:
Please limit PRs to a single topic. If this name needs to be updated it can be, but it should be in a PR dedicated to that cleanup and not part of one that's adding record support.

```suggestion
    Client for making calls to Akamai Fast DNS API using Python Requests
```